### DISCUSSION TITLE
Stay True

### DISCUSSION THREAD

**Discussion by arzadu** [lukebemish/excavated_variants#107] [Q&A] [answered]
Hiya! I've seen in some issues that stay true resources tend not to play nice with excavated variants + mods that generate ores in the end. Does anyone have solutions to these issues other than removing the resource pack entirely? I really enjoy both stay true and excavated variants, and if there's a way to make them work together I would be thrilled.

Asking this over here instead of as an issue because it's not an issue with excavated variants, but folks here might know how to resolve it.

Thanks so much!

**Comment by lukebemish** [comment, marked as answer]
Currently, there's not a ton of ways around this. I'm working on some systems for DynAssetGen that will help mitigate this, but they're a ways out since they require some stuff to be rethought. Here's the issue at present: stay true provides a new end stone texture. Mods with end ores have the ores against the vanilla end stone. As such, DynAssetGen can no longer properly extract the texture bits from the full ore textures when Stay True is present since the backgrounds don't match any more. I can't just always use vanilla/mod textures, as this would break compatibility with texture packs that provide both an ore and background.

The only real mitigation I know of for this, at present, is to provide textures (with a resource pack or the like) that provide the end ore textures with a Stay True style background.

**Reply by lukebemish** [reply]
The other option, if you don't care about variants being present, is to remove or blacklist the end variants (either the stones, original ores, or generated ores). Instructions for doing this with a blacklist filter can be found on the README.

**Reply by arzadu** [reply]
thanks a lot! i'll look into making some modified textures to use.
